src/db/models/User.js

```diff
@@ -1,16 +1,26 @@
-export default (sequelize, DataTypes) => {
+module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define('User', {
-    firstName: {
+    first_name: {
       type: DataTypes.STRING,
-      allowNull: false,
+      allowNull: true,
+    },
+    last_name: {
+      type: DataTypes.STRING,
+      allowNull: true,
     },
-    lastName: {
+    username: {
       type: DataTypes.STRING,
       allowNull: false,
+      unique: true,
     },
     email: {
       type: DataTypes.STRING,
       allowNull: false,
+      unique: true,
+    },
+    password: {
+      type: DataTypes.STRING,
+      allowNull: false,
```

Comment:
It seems like the fields in the migration file does not match what we have in this model.

Reply:
Yes. Apparently, I wrongly assumed that once it was possible to have them as null values in the database, they were not supposed to be present in the model. I will take a second look at it

Reply:
these differences have equally been resolved